src/Symfony/Bundle/SecurityBundle/Tests/Security/FirewallMapTest.php

```diff
@@ -57,10 +58,11 @@ public function testGetListeners()
     {
         $request = new Request();
 
+        $exceptionListener = $this->getMockBuilder(ExceptionListener::class)->disableOriginalConstructor()->getMock();
         $firewallContext = $this->getMockBuilder(FirewallContext::class)->disableOriginalConstructor()->getMock();
         $firewallContext->expects($this->once())->method('getConfig')->willReturn('CONFIG');
-        $firewallContext->expects($this->once())->method('getListeners')->willReturn('LISTENERS');
-        $firewallContext->expects($this->once())->method('getExceptionListener')->willReturn('EXCEPTION LISTENER');
+        $firewallContext->expects($this->once())->method('getListeners')->willReturn(array('LISTENERS'));
```

Comment:
To fulfil the contract `array('LISTENERS')` should be an array of `ListenerInterface` instances